pages/common/fping.md

```diff
@@ -0,0 +1,15 @@
+# fping
+
+> A more powerful ping which ping multiple hosts
```

Comment:
"which ping" --> "which can ping". Also, you need a period at the end of the description.